src/core_mymalloc.h

```diff
@@ -9,7 +9,9 @@ extern "C" {
     extern void *mycalloc(const size_t count, const size_t size);
     extern void *myrealloc(void *p, size_t n);
     extern void myfree(void *p);
+#ifdef VERBOSE_MEMORY    
```

Comment:
Is this meant to be `VERBOSE_MEMORY` rather than just `VERBOSE`?

Reply:
Oops - fixed